### ISSUE TITLE
Gear icon in "list signal sounds" doesn't do anything

### ISSUE THREAD

**Issue by roblourens** [microsoft/vscode#205681]
Testing #205384

- Run "List Signal Sounds"
- Two of these have gear icons, no tooltip, I don't know what it does and clicking it doesn't do anything

![Image](https://github.com/microsoft/vscode/assets/323878/1d8c9d60-f6f6-44a4-8901-842f96b7f8cf)

Also I don't know why the format option says "(undefined)"

**Comment by karthiknadig** [comment]
This woks on latest. Takes you to user settings.json

**Comment by roblourens** [comment]
There's still no tooltip on the button. And why do these settings have it but the others don't?

**Comment by meganrogge** [comment]
They're special - they have values of `userGesture`, `always` and `never`

**Comment by meganrogge** [comment]
I'm setting the tooltip here 

https://github.com/microsoft/vscode/blob/a6671b6ce40a75438455b10e0d888d49203bfc55/src/vs/workbench/contrib/accessibilitySignals/browser/commands.ts#L111

@TylerLeonhardt any thoughts?

@roblourens I don't think this needs to be reopened bc it lacks a tooltip. This is primarily a feature for screen reader users, for whom tooltips aren't even useful.

**Comment by TylerLeonhardt** [comment]
opened an issue about the quick pick tooltips here: https://github.com/microsoft/vscode/issues/206039
